nixos/modules/services/web-apps/alist.nix

```diff
@@ -0,0 +1,313 @@
+{
+  config,
+  lib,
+  pkgs,
+  utils,
+  ...
+}:
+let
+  cfg = config.services.alist;
+  settingsFormat = pkgs.formats.json { };
+in
+{
+
+  meta = {
+    maintainers = with lib.maintainers; [ moraxyc ];
+  };
+
+  options = {
+    services.alist = {
+      enable = lib.mkEnableOption "alist, a file list program";
+      debug = lib.mkEnableOption "debug mode of alist";
+      user = lib.mkOption {
+        type = lib.types.str;
+        default = "alist";
+        description = "Alist user name.";
+      };
+
+      group = lib.mkOption {
+        type = lib.types.str;
+        default = "alist";
+        description = "Alist group name.";
+      };
+
+      dataDir = lib.mkOption {
```

Comment:
Instead of using `dataDir`, how about using `stateDir` like pretty much many many other services?